factions/mummies/units/obelisk/obelisk.xml

```diff
@@ -1,125 +1,134 @@
+<unit>
+	<parameters>
+		<size value="3" />
+		<height value="5" />
+		<ai-build-size value="4"/>
+		<max-hp value="6000" regeneration="0"/>
+		<max-ep value="0" />
+		<armor value="0" />
+		<armor-type value="wood"/>
+		<sight value="5" />
+		<time value="100" />
+		<multi-selection value="true"/>
+		<uniform-selection value="true"/>
+		<cellmap value="true">
+			<row value="010"/>
+			<row value="010"/>
+			<row value="000"/>
+		</cellmap>
+		<levels/>
+		<fields>
+			<field value="land" />
+		</fields>
+		<properties>
+			<property value="burnable"/>
+		</properties>
+		<light enabled="false" />
+		<unit-requirements/>
+		<upgrade-requirements />
+		<resource-requirements>
+			<resource name="gold" amount="100" />
+			<resource name="stone" amount="100" />
+		</resource-requirements>
+		<resources-stored>
+		</resources-stored>
+		<image path="images/obelisk.bmp"/>
+		<image-cancel path="../pyramid/images/tech_cancel.bmp"/>
+		<meeting-point value="true" image-path="../pyramid/images/meeting_point.bmp"/>
+		<selection-sounds enabled="true">
+			<sound path="$COMMONDATAPATH/sounds/tech_click.wav" />
+		</selection-sounds>
+		<command-sounds enabled="true">
+			<sound path="$COMMONDATAPATH/sounds/tech_click.wav" />
+		</command-sounds>
+	</parameters>
+	<skills>
+		<skill>
+			<name value="be_built_skill"/>
+			<ep-cost value="0"/>
+			<speed value="300"/>
+			<anim-speed value="10"/>
+			<animation path="models/obelisk_construction.g3d"/>
+			<sound enabled="false"/>
+			<type value="be_built"/>
+		</skill>
+		<skill>
+			<type value="stop"/>
+			<name value="stop_skill"/>
+			<ep-cost value="0"/>
+			<speed value="300"/>
+			<anim-speed value="100"/>
+			<animation path="models/obelisk.g3d"/>
+			<sound enabled="false"/>
+		</skill>
+		<skill>
+			<type value="produce"/>
+			<name value="produce_skill"/>
+			<ep-cost value="0"/>
+			<speed value="300"/>
+			<anim-speed value="50"/>
+			<animation path="models/obelisk_producing.g3d"/>
+			<sound enabled="false"/>
+		</skill>
+		<skill>
+			<type value="upgrade"/>
+			<name value="upgrade_skill"/>
+			<ep-cost value="0"/>
+			<speed value="300"/>
+			<anim-speed value="60"/>
+			<animation path="models/obelisk_producing.g3d"/>
+			<sound enabled="false"/>
+		</skill>
+		<skill>
+			<type value="die"/>
+			<name value="die_skill"/>
+			<ep-cost value="0" />
+			<speed value="300"/>
+			<anim-speed value="20" />
+			<animation path="models/obelisk_dying.g3d"/>
+			<sound enabled="true" start-time="0">
+				<sound-file path="$COMMONDATAPATH/sounds/tech_building_fall5.wav"/>
+			</sound>
+			<fade value="false"/>
+		</skill>
+	</skills>
+	<commands>
+		<command>
+			<type value="produce"/>
+			<name value="produce_mummy" />
+			<image path="../mummy/images/daemon.bmp"/>
+			<unit-requirements/>
+			<upgrade-requirements/>
+			<produce-skill value="produce_skill"/>
+			<produced-unit name="mummy"/>
+		</command>
+		<command>
+			<type value="produce"/>
+			<name value="produce_giant_mummy" />

```

Comment:
@Robotkiller001 Any reason you decided to add that here instead of giving the regular mummy the ability to morph?

Reply:
bc it wouldn't get the max units possible most of the times by morphing them, the cpu doesn't really like morphing units.